test/commands/status.test.ts

```diff
@@ -1,5 +1,7 @@
 import { expect, test } from "@oclif/test";
 import { NETWORKS } from "../../src/config";
+process.env.DEV_MODE = "false";
```

Comment:
You should not have to deal with it here :/ If DEV_MODE breaks your tests disable it